Foxtrot/Driver/Rewriting/PostRewriteChecker.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Compiler;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Contracts.Foxtrot
+{
+    public sealed class PostRewriteChecker : Inspector
+    {
+        private Member currentMember;
+        private readonly Action<CompilerError> handleError;
+
+        public PostRewriteChecker(Action<CompilerError> handleError)
+        {
+            this.handleError = handleError;
+        }
+
+        public override void VisitProperty(Property property)
+        {
```

Comment:
Does it means that each `Visit` method can simply return if `this.currentMember` is not `null`?